### PR TITLE
Makes SM sliver appear on the SM

### LINKED ISSUE #2694: SM layer issue
SM sliver appears under the actual SM.

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request

closes #2694

## Why It's Good For The Game

It's super confusing to have it appear under the crystal.

## Changelog
:cl:
tweak: SM sliver appears on the SM again.
/:cl:

### CODE REVIEW

File: code/game/objects/items/theft_tools.dm [hunk 1]
```diff
@@ -11,6 +11,7 @@
 	icon_state = "plutonium_core"
 	item_state = "plutoniumcore"
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF
+	layer = 5
```
Comment (PowerfulBacon) [inline code/game/objects/items/theft_tools.dm]: Use the defines for layers rather than a magic number

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]

File: code/game/objects/items/theft_tools.dm (hunk 1 above)
Comment (Archanial) [reply code/game/objects/items/theft_tools.dm]: sure, I'll change the SM layer too.

**Comment by PowerfulBacon** [comment]
Does this work, or would you need to have the shard slightly above the crystal?

**Comment by Archanial** [comment]
this works

**Review by PowerfulBacon — APPROVED** [review]
I'll take your word for it

**Pull request merged**